docs/self-hosted/docker-guide.md

```diff
@@ -149,6 +149,51 @@ docker compose up
 The specified image will be pulled and the containers recreated. Migrations will happen automatically so once the
 containers have started you will be on the latest version (or the version you specified).
 
+## Sample Dockerfile
+
+If you need to build a Docker image on top of the base docker image, here's a sample `Dockerfile`:
+
+```dockerfile
+FROM directus/directus:10.6.1
+
+USER node
+
+COPY --chown=node .env .
+
+COPY --chown=node ./config/docker/entrypoint.sh .
+
+COPY --chown=node ./extensions ./extensions
+
+EXPOSE 8055
+
+ENTRYPOINT ["./entrypoint.sh"]
+```
+
+where you might be connecting to an external database and file system for uploads.
+
+The `entrypoint.sh` file in `config/docker` (you can choose to put `entrypoint.sh` anywhere) would look like this:
+
+```bash
+#!/bin/sh
+
+npx directus bootstrap
+npx directus start
+
+exec "$@"
+```
```

Comment:
```suggestion
```

I wouldnt recommend overwriting the entrypoint but if you must do so then use `node /directus/cli.js bootstrap && node /directus/cli.js start` (which is already the current entrypoint) because `npx ...` has a noticeable overhead when starting up the docker.